frame/support/procedural/src/pallet/parse/mod.rs

```diff
@@ -62,6 +66,25 @@ pub struct Def {
 }
 
 impl Def {
+	fn mark_benchmarking_cached_calls(storage_def: &mut StorageDef, item: &mut syn::Item) {
```

Comment:
This code is also way to overcomplicated.

```
if attr.path.is_ident("benchmarking") { attr.parse_args::<keyword::caching>()? }
```

Reply:
This is great, thanks! The docs on `parse_args` are extremely sparse and I was never able to get it working